Resources/Prototypes/Corvax/Roles/Jobs/Syndicate/Smuggler.yml

```
@@ -0,0 +1,47 @@
+  canBeAntag: true
+  icon: "JobIconSyndicate"
+  supervisors: job-supervisors-nobody
+  weight: 40
+  displayWeight: 40
+  setPreference: true
+  access:
+  - GeneraSyndicatelAccess
+  accessGroups:
+  - GeneraSyndicatelAccess
+  special:
+  - !type:AddComponentSpecial
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)